### PR TITLE
init

### PR DESCRIPTION
**Pull request by incogitance**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const app = express();
+const PORT = process.env.PORT || 3000;
+// TODO - implement solution for backend
+
+// Create GET API endpoint
+// TODO - Create all database connections
+// TODO -
+
+// Serve the public folder for client
+app.use(express.static(__dirname + '/../public'));
```
Comment (WarrenMfg) [inline server/index.js:13]: __dirname works here? I couldn't get it to work for me...

**Review by WarrenMfg — APPROVED** [review]
Looks great!

File: server/index.js (hunk 1 above)
Comment (incogitance) [reply server/index.js:13]: it certainly worked for me...

**Review by WarrenMfg — APPROVED** [review]
I was showing that I approved these. But somehow it didn't go through.

**Review by JohnCharamonde — APPROVED** [review]
lgtm

**Pull request merged**